### DISCUSSION TITLE
"Loading..." as name of Item in Side Navigation

### DISCUSSION THREAD

**Discussion by jordanberry** [microsoft/Microsoft-Fabric-workload-development-sample#134] [Help]
When selecting an Item created with our Workload, it appears that the item in the Side Navigation does not resolve even though the Get Item call to our Backend returns the item payload. (You can see in the main page, the JSON representation of this Item). 

Any thoughts on why this may be happening or if there is something additional needed to update the Item Name in the Side Navigation? 

<img width="1509" alt="Screenshot 2024-10-20 at 3 57 21 PM" src="https://github.com/user-attachments/assets/4931f8ce-a799-4a17-a80a-e85af971ab06">

**Comment by ghost** [comment]
I guess we are also not able to close the item from the side panel once we open them. Which is preventing us to switch between the workload items, without refreshing the page.

**Comment by michal-mano** [comment]
Hi, yes, this is a known issue and a fix has already been sent out. This should resolve itself within a week or two on your end, no need to update anything. As @vinayslumel mentioned, you can refresh the page and then you'll be able to click "x" in the top right of the multitasking cell in order to switch between items.
